server.elm

```diff
@@ -2824,8 +2852,61 @@ lastEditScript = """
       document.body.addEventListener('mouseover', linkModeHover1, false);
       document.body.addEventListener('mouseout', linkModeHover2, false);
     }
+
+    
+
+    function copy_website(source, dest) {
```

Comment:
Please add a comment on this function. Does it take folders? Should it contain in dest the name of the folder to be copied?